client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -45,23 +47,22 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 							{ translate( 'Everything you need to build and grow your site from any device.' ) }
 						</h3>
 					</div>
-					<div className="go-mobile__app-badges">
```

Comment:
Refactored to prevent an empty `<div className="go-mobile__app-badges">` rendering when the app badges don't display.